core/runjob_test.go

```diff
@@ -62,6 +63,7 @@ func (s *SuiteRunJob) TestRun(c *C) {
 	time.Sleep(200 * time.Millisecond)
 	container, err := job.getContainer()
 	c.Assert(err, IsNil)
+	c.Assert(container.Config.Entrypoint, DeepEquals, []string{"/bin/bash", "-c"})
```

Comment:
We would need two test cases for this, one with no Entrypoint provided to make sure that default value is honoured and another with entrypoint to make sure that default one is overwritten

Reply:
@taraspos sure, I will add the tests once I got the chance 👌